Sources/SpelledPitch.swift

```diff
@@ -1,12 +1,12 @@
 //
-//  Tone.swift
+//  Pitch.swift
 //  MusicNotationCore
 //
 //  Created by Kyle Sherman on 6/15/15.
 //  Copyright © 2015 Kyle Sherman. All rights reserved.
 //
 
-public struct Tone {
+public struct SpelledPitch {
```

Comment:
Since now the term `SpelledPitch` is a little less understandable by people who are not aware, let's put a little documentation comment for the struct above here. Say something like 

> A pitch indicates the note letter (a-g), octave, and accidental. Since certain pitches are the same, but can be shown differently (like a flat and b sharp), the "spelling" of the pitch is important. This struct represents a pitch that is already in the desired spelling. So, this assumes the user of this struct knows which spelling to pick.

Reply:
Good idea. done